### DISCUSSION TITLE
窗口大小和位置怎么回到默认？

### DISCUSSION THREAD

**Discussion by NiMing561** [Meloong-Git/PCL#3158] [提问] [answered]
刚才我遇到了一个问题，我发现鼠标原来可以把最新版PCL的窗口边缘拖大，但拖走之后回不去了（我有强迫症），请各位大神教教怎么把窗口大小恢复默认吧（悲

**Comment by allMagicNB** [comment]
删注册表

**Comment by Light-Beacon** [comment, marked as answer]
后缀名改成`.reg`然后双击导入注册表就行
[pcl_window_size_reset.txt](https://github.com/Hex-Dragon/PCL2/files/13991306/pcl_window_size_reset.txt)
